src/components/bulkConcepts/container/BulkConceptsPage.jsx

```diff
@@ -66,18 +85,43 @@ export class BulkConceptsPage extends Component {
     }
   };
 
+  getBulkConcepts = () => {
+    const { currentPage, fetchBulkConcepts: bulkConceptsFetched } = this.props;
+    bulkConceptsFetched(currentPage);
+  }
+
   handleChange = (event) => {
     const {
       target: { value },
     } = event;
     this.setState(() => ({ searchInput: value }));
   };
 
-  handleSearch = (event) => {
-    event.preventDefault();
+  searchOption = () => {
     const { searchInput } = this.state;
     const query = `q=${searchInput.trim()}`;
-    this.props.fetchFilteredConcepts('CIEL', query);
+    const { currentPage, fetchFilteredConcepts: fetchedFilteredConcepts } = this.props;
+    fetchedFilteredConcepts('CIEL', query, currentPage);
```

Comment:
Don't we have a constant for the 'CIEL' literal string?

Reply:
This function was already in existence on the codebase. My task was to implement pagination. I only refactored so that I can check if I should load the bulk concept or the searched concept